nautobot/core/templates/components/panel/body_content_text.html

```diff
@@ -1,6 +1,10 @@
 {% load helpers %}
-{%  if enable_markdown %}
-    {{ text_content | render_markdown }}
+{%  if render_as == render_options.json %}
+    {{ text_content | placeholder | render_json }}
+{% elif render_as == render_options.yaml %}
+    {{ text_content | placeholder | render_ysml }}
+{% elif render_as == render_options.markdown %}
+    {{ text_content | placeholder | render_markdown }}
```

Comment:
Do these work correctly for empty values? I'm wondering if the `placeholder` and `render_*` are in the correct order here or if they need to be swapped.

Actually, for `json` and `yaml` cases we probably don't need the `placeholder` at all, since empty/falsey values are valid and meaningful for those datatypes.

Also, looking at our existing usage, the `json` and `yaml` cases probably should be wrapped in `<pre>...</pre>` - maybe the plaintext case as well?

Reply:
Placeholder just displays the dash sign wrapped in html. 

Json has built-in wrapping code into `<pre>` tags.

I will try to figure something better for this placeholder handling.